src/settings/Validation/engine/validators.js

```diff
@@ -48,4 +49,8 @@ export default {
     validate: isOverdueFindGreaterThan,
     message: <FormattedMessage id="ui-circulation.settings.validate.greaterThanZero" />,
   },
+  isItemLimitIsInInterval: {
+    validate: isInIntervalOrNull.bind(null, 1, 9999),
```

Comment:
```suggestion
    validate: value => isInIntervalOrNull(1, 9999, value),
```
Arrow functions like `isInIntervalOrNull` does not have context, so using `bind` of them can be misleading. I would rather simplify the mental work with the suggested code above.